scene/gui/text_edit.cpp

```diff
@@ -7499,7 +7494,7 @@ void TextEdit::_scroll_down(real_t p_delta) {
 		if (target_v_scroll > max_v_scroll) {
 			target_v_scroll = max_v_scroll;
 		}
-		if (Math::abs(target_v_scroll - v_scroll->get_value()) < 1.0) {
+		if (Math::abs(target_v_scroll - v_scroll->get_value()) < 1.0 || !p_animate) {
```

Comment:
```suggestion
		if (!p_animate || Math::abs(target_v_scroll - v_scroll->get_value()) < 1.0) {
```
Put cheaper condition first